web3/_utils/contracts.py

```diff
@@ -265,6 +272,16 @@ def get_fallback_function_info(
     return fn_abi, fn_selector, fn_arguments
 
 
+def get_receive_function_info(
+    contract_abi: ABI=None, fn_abi: ABIFunction=None
+) -> Tuple[ABIFunction, HexStr, Sequence[Any]]:
```

Comment:
We have a general convention of *strict* typing for return values and *generic* typing for inputs.

- *generic* types for inputs
    - `Sequence`, `Collection`, `Iterable`
- *concrete* types for return values
    - `List`, `Tuple`, etc

Unless I'm missing something, I believe that the `Sequence` here should be `Tuple[Any, ...]` since it is a return value.